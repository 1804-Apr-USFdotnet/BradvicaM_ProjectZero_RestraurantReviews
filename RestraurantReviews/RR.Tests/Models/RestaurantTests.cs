using System;
using ApprovalTests;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using RR.Models;

namespace RR.Tests.Models
{
    [TestClass]
    public class RestaurantTests
    {
        [TestMethod]
        [UseReporter(typeof(DiffReporter))]
        public void Restaurant_OnCall_DisplaysProperly()
        {
            var restaurant = new Restaurant
            {
                Address = new Address
                {
                    City = "Kansas City",
                    State = "MO",
                    Street = "123 Barbeque St.",
                    ZipCode = 81721
                },
                Id = Guid.Empty,
                Name = "Billy Bobs Texas BBQ",
                PhoneNumber = "8761234121",
                Rating = 0.0,
                Website = "www.billybobs.com"
            };

            Approvals.Verify(restaurant);
        }
    }
}
