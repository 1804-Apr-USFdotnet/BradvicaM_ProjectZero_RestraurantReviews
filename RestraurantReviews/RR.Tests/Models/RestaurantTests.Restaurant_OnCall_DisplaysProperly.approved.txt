
Id: 00000000-0000-0000-0000-000000000000
Name: Billy Bobs Texas BBQ 
Address: 123 Barbeque St. Kansas City MO 81721
PhoneNumber: 8761234121
Website: www.billybobs.com
Rating: 0